.github/workflows/release_windows_pytorch_wheels.yml

```diff
@@ -78,11 +78,23 @@ permissions:
 
 jobs:
   release:
-    name: Release | ${{ inputs.amdgpu_family }} | py ${{ matrix.python_version }}
+    name: Release | ${{ inputs.amdgpu_family }} | py ${{ matrix.python_version }} | torch ${{ matrix.pytorch_version }}
     strategy:
       fail-fast: false
       matrix:
         python_version: ["3.11", "3.12", "3.13"]
+        # TODO: switch to 'nightly' to match our Linux workflows?
+        # TODO(#1677): Get 2.8 and 2.9 builds working and enable them
+        # pytorch_version: ["release/2.8", "release/2.9", "main"]
+        pytorch_version: ["main"]
+        include:
+          # TODO(#1677): Get 2.8 and 2.9 builds working and enable them
+          # - pytorch_version: release/2.8
+          #   pytorch_patchset: rocm_2.8
+          # - pytorch_version: release/2.9
+          #   pytorch_patchset: rocm_2.9
+          - pytorch_version: main
+            pytorch_patchset: main
```

Comment:
```suggestion
          - pytorch_version: nightly
            pytorch_patchset: nightly
```

Reply:
As above, let's keep main until Scott is back and address this separately.